cyberdrop_dl/crawlers/beeg.py

```diff
@@ -43,6 +43,10 @@ def __post_init__(self) -> None:
         self.request_limiter = AsyncLimiter(4, 1)
 
     async def video(self, scrape_item: ScrapeItem, video_id: str) -> None:
+        canonical_url = PRIMARY_URL / "videos" / video_id
```

Comment:
This is not a valid URL format. It always returns 404 on the site

Reply:
This works but `video_id` needs to be casted as `int` to remove 0 padding
```suggestion
        canonical_url = PRIMARY_URL / video_id
```